src/Components/CloseEmailButton/index.jsx

```diff
@@ -0,0 +1,12 @@
+import { Link } from "react-router-dom"
+
+function CloseEmailButton (){
+    return (
+        <div className="grid justify-items-end pt-4">
```

Comment:
Having this container here also limits the usability of this component - I would remove it from here and instead put this code (just this div containing the button) into your OpenEmails page instead.

That way you can use this component anytime you need a button that links to a page.